test/conf/protractor.local.android.conf.js

```diff
@@ -24,16 +22,17 @@ config.capabilities = {
 
 config.plugins = [
 	{
-		path: join(process.cwd(), './build/index.js'),
+		path: resolve(process.cwd(), './build/index.js'),
 		options: {
-			baselineFolder: join(process.cwd(), './localBaseline'),
+			baselineFolder: resolve(process.cwd(), './test/images/baseline'),
+			screenshotPath: resolve(process.cwd(), './test/images/'),
+			formatImageName: '{tag}-{logName}-{width}x{height}',
 			debug: false,
-			formatImageName: `{tag}-{logName}-{width}x{height}`,
-			screenshotPath: join(process.cwd(), '.tmp/'),
 			savePerInstance: true,
 			autoSaveBaseline: true,
 			blockOutStatusBar: true,
 			blockOutToolBar: true,
+			clearRuntimeFolder: true
```

Comment:
This needs to be removed, I also need to remove this from the module because it removes it per instance, not in the  `onPrepare` / `onComplete`